### PR TITLE
restore setPosition overloading

### PR DESCRIPTION
**Pull request by ianhi**
fix: #79

### CODE REVIEW

File: pymmcore_plus/core/_mmcore_plus.py [hunk 1]
```diff
@@ -432,9 +433,26 @@ def getZPosition(self) -> float:
...
+        """Set position of the stage in microns."""
+        n_args = len(args)
+        if n_args == 1:
+            return super().setPosition(args[0])
+        elif n_args == 2:
+            return super().setPosition(args[0], args[1])
+        else:
+            raise ValueError(
+                f"setPosition takes exactly one or two arguments. Got {len(args)}"
+            )
```
Comment (tlambert03) [inline pymmcore_plus/core/_mmcore_plus.py:455]:
```suggestion
        return super().setPosition(*args)
```

Comment (tlambert03) [reply pymmcore_plus/core/_mmcore_plus.py:455]: we don't need to raise the value error here.  super() will do it

Comment (ianhi) [reply pymmcore_plus/core/_mmcore_plus.py:455]: oooh

**Comment by ianhi** [comment]
Thought: we should add a test that the typing of this library is a superset of the pymmcore types. Not quite sure how to implement that, but would prevent similar issues in the future

**Comment by tlambert03** [comment]
Testing typing is rather annoying.  One can use pytest-mypy-plugins to do it (and I do in other libraries to make sure types are working).  But it's slow, and takes a lot of work to keep it working as mypy versions change.  Not worth it for something like this, I should have just checked the super sig

**Pull request merged**